compiler/wasm/package.json

```diff
@@ -29,9 +29,10 @@
     "url": "https://github.com/noir-lang/noir/issues"
   },
   "scripts": {
-    "build": "WASM_OPT=$(bash -c ./wasm-opt-check.sh) webpack",
+    "install:wasm_pack": "./scripts/install_wasm-pack.sh",
+    "build": "yarn install:wasm_pack && WASM_OPT=$(bash -c ./scripts/wasm-opt-check.sh) webpack",
```

Comment:
Running this on every builds seems to be a bit of an oof. Can we move this to a `bootstrap` step?

Reply:
The point was to allow upstream scripts to call `yarn build` and forget about any implementation details (as they do now with install_wasm-bindgen, for example), relying on the particular callee (cargo in this case) for caching. The impact is negligible if I have it already installed in my system.

I can change it, but it would also impact `aztec-packages` if this particular package has an extra step.

Reply:
Furthermore, the idea is to avoid this situation: if you search `aztec-packages` for the string "install_wasm-bindgen", you get 5 hits. 3 of them point to a script in `.github/scripts`, the other two to `noir/scripts`. Both scripts AFAIK do the same thing, but do it differently...

Unless this slows down CI considerably, I would try to make each package able to build itself in isolation. That or we create an universal "base-env-script" or docker image that has every common dependency and is able to set the stage to build everything...but sprinkling magic installs through 2 repos in different scripts is a recipe for chaos IMHO.

Reply:
Yeah, I probably should have deleted this once I added the other review comment. With cargo-binstall, this is cheap enough that we could install on every build.

Agreed that current state is best plan.